doc/ChangeLog

```diff
@@ -1,4 +1,71 @@
 ===============================================================
+Tag name: ctsm5.1.dev171
+Originator(s): olyson (Keith Oleson,UCAR/TSS)
+Date: Fri 01 Mar 2024 03:27:14 PM MST
+One-line Summary: Set initial t_soisno=272 for soils and 274K for urban road
+
+Purpose and description of changes
+----------------------------------
+
+ Issue #2338 and PR #2355 explain:
+ Soil temperature initialization not implemented correctly in
+ ctsm5.1.dev058 and thus subsequent tags.
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_1
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Bugs fixed
+----------
+CTSM issues fixed (include CTSM Issue #):
+ Fixes #2338
+
+Notes of particular relevance for developers:
+---------------------------------------------
+Changes to tests or testing:
+ SMS_Lm3_D_Mmpi-serial.1x1_brazil.I2000Clm50FatesCruRsGs.izumi_intel.clm-FatesColdHydro
+ added to expected failures, issue #2373, to be revisited when #2384 is
+ resolved.
+
+Testing summary:
+----------------
+ [PASS means all tests PASS; OK means tests PASS other than expected fails.]
+
+  regular tests (aux_clm: https://github.com/ESCOMP/CTSM/wiki/System-Testing-Guide#pre-merge-system-testing):
+
+    derecho -----
+    izumi -------
+
+Answer changes
+--------------
+
+Changes answers relative to baseline: YES
+
+  Summarize any changes to answers, i.e.,
+    - what code configurations: all
+    - what platforms/compilers: all
+    - nature of change: roundoff
```

Comment:
I would say larger than roundoff but not climate-changing.  The original diagnostics for this change are here:
https://webext.cgd.ucar.edu/I2000/ctsm51c6_PPEn08ctsm51d023_2deg_GSWP3V1_Sparse400_cs_ts_tsoisno272_2000AD/lnd/ctsm51c6_PPEn08ctsm51d023_2deg_GSWP3V1_Sparse400_cs_ts_tsoisno272_2000AD.381_400-ctsm51c6_PPEn08ctsm51d023_2deg_GSWP3V1_Sparse400_cs_ts_2000AD.381_400/setsIndex.html

Reply:
Thank you, Keith.